scipy/sparse/tests/test_sparsetools.py

```diff
@@ -53,17 +53,17 @@ def test_regression_std_vector_dtypes():
     # Regression test for gh-3780, checking the std::vector typemaps
     # in sparsetools.cxx are complete.
     for dtype in supported_dtypes:
-        ad = matrix([[1, 2], [3, 4]]).astype(dtype)
+        ad = np.array([[1, 2], [3, 4]]).astype(dtype)
         a = csr_matrix(ad, dtype=dtype)
 
         # getcol is one function using std::vector typemaps, and should not fail
-        assert_equal(a.getcol(0).todense(), ad[:,0])
+        assert_equal(a.getcol(0).toarray(), ad[:, [0]])
 
 
 @pytest.mark.slow
 @pytest.mark.xfail_on_32bit("Can't create large array for test")
 def test_nnz_overflow():
-    # Regression test for gh-7230 / gh-7871, checking that coo_todense
+    # Regression test for gh-7230 / gh-7871, checking that coo_toarray
```

Comment:
This may not be an intentional change?

Reply:
The code below didn't invoke `todense` also before the change (line 79), so this is a fix of the comment, and it's intentional.